pkgs/applications/video/mpv/scripts/mpris.nix

```diff
@@ -1,28 +1,42 @@
-{ stdenv, fetchFromGitHub, pkgconfig, gobject-introspection, mpv }:
+{ stdenv, fetchpatch, fetchFromGitHub, pkgconfig, gobject-introspection, mpv }:
 
 stdenv.mkDerivation rec {
-  name = "mpv-mpris-${version}.so";
-  version = "0.4";
+  pname = "mpv-mpris";
+  version = "0.5";
 
   src = fetchFromGitHub {
     owner = "hoyon";
     repo = "mpv-mpris";
     rev = version;
-    sha256 = "1fr3jvja8s2gdpx8qyk9r17977flms3qpm8zci62nd9r5wjdvr5i";
+    sha256 = "07p6li5z38pkfd40029ag2jqx917vyl3ng5p2i4v5a0af14slcnk";
   };
+  # Enables to "make SCRIPTS_DIR=... install" https://github.com/hoyon/mpv-mpris/pull/38
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/hoyon/mpv-mpris/pull/38.patch";
```

Comment:
```suggestion
      url = "https://github.com/hoyon/mpv-mpris/commit/5e00f275a666cc1aea1e93ae46f888293c6e502d.patch";
```

Reply:
The pull request might change. The commit won't.

Reply:
:+1:. Didn't imagine the commit URL won't disappear ever.

Reply:
I mean, for e.g, the previous last commit of this PR is still alive: https://github.com/NixOS/nixpkgs/commit/595ad51476b3847a46eff4c638aa2635a8be1d8f.patch

Reply:
github makes them immutable.